crates/test-modules/modules/crates/resource-main/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "resource-main"
+version = "0.1.0"
+authors = ["Peter Huene <peter@huene.dev>"]
+edition = "2018"
```

Comment:
Mind throwing in `publish = false` for this (and other crates now that I think about it, but that can happen somewhere else of course)

Reply:
I'll add that to each test module.